packages/react/src/components/Button/Button.js

```diff
@@ -143,19 +143,33 @@ const Button = React.forwardRef(function Button(
   let otherProps = {
     disabled,
     type,
+    'aria-describedby': kind === 'danger' ? 'danger-description' : null,
```

Comment:
I think we need to check if `kind` is `danger`, `danger--tertiary`, or `danger--ghost`. During testing, I only get the `danger` warning for the first button in the `Danger Button` story